picking_dispatch/stock.py

```diff
@@ -92,7 +92,7 @@ def do_partial(self, cr, uid, ids, partial_datas, context=None):
                 dispatch_id = unfinished_dispatch_ids[dispatch_id]
             maybe_finished_dispatches.setdefault(
                 dispatch_id, []).append(move.id)
-        for dispatch_id, move_ids in maybe_finished_dispatches.iteritems():
+        for dispatch_id, move_ids in maybe_finished_dispatches.iteritems(): # pylint: disable=W1620
```

Comment:
You are right!
But you aren't use the "optimized" way.
A iterator is good when you use a part of the all items.
But here you are using the full item.